cloud_sql/database_basic/main.tf

```diff
@@ -30,5 +30,5 @@ resource "google_sql_database_instance" "instance" {
     tier = "db-f1-micro"
   }
 
-  deletion_protection = false # set to true to prevent destruction of the resource
+  deletion_protection = false # set `deletion_protection_enabled` flag to true to enable deletion protection of an instance at the GCP level. Enabling this protection will guard against accidental deletion across all surfaces (API, gcloud, Cloud Console and Terraform) by enabling the GCP Cloud SQL instance deletion protection. On the other hand, `deletion_protection` flag prevents destroy of the resource only when the deletion is attempted in terraform.
```

Comment:
Let's move this comment to above `deletion_protection`. Also, please wrap at ~80 characters as this shows up in the docs and horizontal scrolling is painful for our customers.

Reply:
Done